deploy/example_catalog/cr-app-cdh632cm.json

```diff
@@ -0,0 +1,415 @@
+                    "name": "HBase REST"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 9095,
+                    "path": "/"
+                },
+                "id": "hbase-thrift",
+                "label": {
+                    "name": "HBase Thrift Server"
+                }
+            },
+            {
+                "id": "hive-metastore",
+                "label": {
+                    "name": "HIVE Metastore"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 10002,
+                    "path": "/"
+                },
+                "id": "hive-server",
+                "label": {
+                    "name": "HIVE Server 2"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": false,
+                    "port": 50111
+                },
+                "id": "webhcat",
+                "label": {
+                    "name": "WebHCat Server"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 8888,
+                    "path": "/"
+                },
+                "id": "hue",
+                "label": {
+                    "name": "Hue Console"
+                }
+            },
+            {
+                "endpoint": {
+                    "isDashboard": true,
+                    "urlScheme": "http",
+                    "port": 18088,
+                    "path": "/"
+                },
+                "id": "spark-history",
+                "label": {
+                    "name": "Spark history server"
+                }
+            },
+            {
+                "id": "impala",
+                "label": {
+                    "name": "Impala service"
+                }
+            },
+            {
+                "id": "oozie",
+                "label": {
+                    "name": "OOZIE service"
+                }
+            }
+        ],
+        "defaultConfigPackage":  {
+            "packageURL": "file:///opt/configscripts/appconfig.tgz"
+        },
+        "roles": [
+            {
+                "id": "cmserver",
+                "cardinality": "1"
+            },
+            {
+                "id": "primary",
+                "cardinality": "1"
+            },
+            {
+                "id": "hbase",
+                "cardinality": "1"
+            },
+            {
+                "id": "hive",
+                "cardinality": "1"
+            },
+            {
+                "id": "impala",
+                "cardinality": "1"
+            },
+            {
+                "id": "apps",
```

Comment:
do we allow apps to be 0 in epic or no ?

Reply:
@kmathur2 Cardinality for apps is 1 subject to if HA is selected. In Kubedirector, we will keep it as 0+ to keep it optional for the user while creating the cluster.